zerver/views/registration.py

```diff
@@ -393,6 +402,7 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
             do_change_password(user_profile, password)
             do_change_full_name(user_profile, full_name, user_profile)
             do_set_user_display_setting(user_profile, "timezone", timezone)
+            do_set_user_display_setting(user_profile, "default_language", default_language)
```

Comment:
Is there a reason why we use `do_set_user_display_setting(user_profile, "default_language", default_language)` instead of `do_change_user_default_language(user_profile, default_language)`?

Reply:
I realized that there was actually no use for a special function to change the user's default language, so I just have removed that function and have kept this only.